releases.json

```diff
@@ -21,7 +21,7 @@
         {
             "device": "gpu",
             "min_version": "1.4.2",
-            "max_version": "1.4.2",
```

Comment:
Could you please add the config to `releases` like:

```
"releases": [
        {
            "device": "gpu",
            "version": "1.4.5",
            "cuda_version": "cu121",
            "os_version": "ubuntu22.04",
            "python_version": "py310",
            "pytorch_version": "2.1.1"
        }
    ]
```

Reply:
Is it necessary to do this for every release? Or only when one of the `__version` fields change?